### PR TITLE
Compose form dynamic color

### PR DESCRIPTION
**Pull request by bivashpandey**
- Compose Form
- Dynamic Color in Form
- New Theme and existing theme with similar brightness
- Color Converter in utils to change the intensity of any hex colour

### CODE REVIEW

**Review by just1ngray — COMMENTED** [review]
Everything seems to be in order. I'll let someone else approve/comment since it may disturb your automated tests.

**Comment by ynaffit96** [comment]
Got a runtime error when I tried to run this

Screenshot (14)

File: client/pages/Compose.js [hunk 1]
```diff
@@ -15,8 +15,10 @@ import Layout from "./components/Layout";
 import axios from "axios";
 import defaults from "../utils/defaults";
 import AutoCompleteText from "./components/AutoCompleteText";
+import styles from "../styles/settings.module.css";
```
Comment (just1ngray) [inline client/pages/Compose.js:18]: We don't use this but whatever 😆

**Review by just1ngray — APPROVED** [review]
Great, this looks much nicer and I like how it pops!

Edit: I discussed this with Tiffany through Teams and she is OK for a merge.

**Pull request merged**